docs/decisions/01-system-design-guidance.md

```diff
@@ -0,0 +1,48 @@
+﻿# System Design Guidance
+
+Priorities for the project, and some guidance on how to balance those priorities when making high level architectural decisions.
+
+## Status
+
+- [ ] Decided (proposal was agreed and adopted)
+- [ ] Decided Against (agreed not to adopt)
+- [ ] Deferred (no decision, but should be revisited later)
+- [ ] Superseded (later events or decisions make this no longer relevant)
+
+# Decision
+
+This is about guidance, so here's the guidance.
+
+1. Prefer multiple providers. There will be system components that can be rented from managed service providers (you know, clouds). It's better to choose technologies that are available in multiple cloud providers. In the future, it would be even better to support multiple services in a category (MySQL AND PostreSQL, for example). That's not feasible now, but try not to design out the option of doing it later. 
+2. Be self-hostable. Don't choose services that are only available in the cloud (like snowflake or dynamodb), or that are difficult to administer (like kafka). This goes double if it's only available from a single provider.
+3. Plan for scale-up. Networks tend to grow over time. It should be a reasonable amount of work for a volunteer admin to add capacity as the need arises.
```

Comment:
Will this mean that our servers need to support horizontally scaling (potentially with an exception for the DB) from the start?

Reply:
I think we should expect to have to scale horizontally, and design to facilitate that. But I think a design that's amenable to horizontal scaling in the future is all we would need for now.